### PR TITLE
DNM Replace direct crontab(s) management in sepia and ovh labs with ansible playbook

### PR DESCRIPTION
**Pull request by yuriw**
@zmc @djgalloway  

Pls review

This is a current working version.  Pls suggest how to improve the structure.

(I'd like to see if it's easy to make `hosts:` as part of `suites:` clause )

http://tracker.ceph.com/issues/15823

Signed-off-by: Yuri Weinstein yweinste@redhat.com

### CODE REVIEW

File: nightlies/ansible-crons/crons.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# *** SEPIA LAB ***
+- hosts: teuthology.front.sepia.ceph.com
+  vars:
+    qa_email: "-e ceph-qa@ceph.com"
+    cron_wrapper: "/home/teuthology/bin/cron_wrapper"
+    sepia_cron_user: "teuthology"
+    rados_command: "/home/teuthology/bin/schedule_rados.sh"
+
+    suites:
+
```
Comment (djgalloway) [inline nightlies/ansible-crons/crons.yaml]: I'd get rid of this spacing so it's clear `suites` and its items, `rgw`, `fs`, and `rados` are items.

    #Example:
      suites:
        rgw:
          name: rgw
          etc.
        fs:
          name: fs
          etc.

File: nightlies/ansible-crons/crons.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      rados:
+        name: rados
+        weekday: 2,7
+        hour: 17
+        min: 15
+        kernel:
+        priority:
+        filter:
+        # this case for rados can me done manually for now
```
Comment (djgalloway) [inline nightlies/ansible-crons/crons.yaml]: Typo

File: nightlies/ansible-crons/crons.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        subset: --subset $(echo "(($(date +%U) % 4) * 7) + CHANGE TO 0...6" | bc)/28
+
+  tasks:
+  - name: "suite {{item.key}} on branch: MASTER"
+    cron:
+      #the line below will be supported in v2.0
+      #cronvar: name="EMAIL" value="yweinste@redhat.com"
+      user: "{{sepia_cron_user}}"
```
Comment (djgalloway) [inline nightlies/ansible-crons/crons.yaml]: I'm being picky but I prefer the formatting `"{{ var_name }}"`

File: nightlies/ansible-crons/crons.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      name: "suite: {{item.value.name}}  branch: master"
+      hour: "{{item.value.hour}}"
+      minute: "#{{item.value.min}}"
```
Comment (djgalloway) [inline nightlies/ansible-crons/crons.yaml]: Won't this comment each line?  Or is that desired?

**Comment by djgalloway** [comment]
Overall, LGTM.  Curious on @zmc's take.

File: nightlies/ansible-crons/crons.yaml (hunk 4 above)
Comment (yuriw) [inline nightlies/ansible-crons/crons.yaml]: `#` is on purpose as when I test it actually writes to a realt crontab

**Comment by liewegas** [comment]
ceph-qa-suite is now merged into ceph.git.  Please resubmit this PR against ceph.git.

**Pull request closed without merging**